libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -62,7 +62,7 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM {
     //  should be called if gyro offsets are recalculated
     void reset_gyro_drift() override;
 
-    void            update(bool skip_ins_update=false) override;
+    void            update(bool skip_ins_update=false, bool run_ekf=true);
```

Comment:
what happened to the override?

Reply:
the override prevent it to compiles, as the function declaration changes, so it normally does not need the override anymore, right?

Reply:
ooooor make that bool check in the function be a runtime variable that you set and dont' pass it in.. such as a new non-volatile frame type